packages/data-context/src/actions/AuthActions.ts

```diff
@@ -72,10 +86,10 @@ export class AuthActions {
       return
     }
 
-    this.setAuthenticatedUser(user as AuthenticatedUserShape)
+    this.setAuthenticatedUser(user)
 
     this.ctx.update((coreData) => {
-      coreData.cancelActiveLogin = null
+      this.cancelActiveLogin = null
     })
```

Comment:
We can loose the wrapping `update` if cancelActiveLogin remains local.

Reply:
+1

Reply:
7572f7811f